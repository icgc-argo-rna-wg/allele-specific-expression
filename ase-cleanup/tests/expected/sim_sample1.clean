contig	position	variantID	refAllele	altAllele	refCount	altCount	totalCount	ase_ratio	ref_bias	AEI_pval	AEI_padj
chr1	100101	.	G	C	15	8	23	0.6521739130434783	0.6179314888010541	0.832212797791076	0.9345662958086165
chr1	100701	.	A	G	12	12	24	0.5	0.5337917521075416	0.8387561049050227	0.9345662958086165
chr1	101701	.	G	C	6	23	29	0.20689655172413793	0.6179314888010541	8.484130101410102e-06	0.00018295331415094085
chr1	101801	.	G	A	3	23	26	0.11538461538461539	0.47317805383022776	0.00022175237205059387	0.00275179712257625
chr1	102101	.	C	A	11	10	21	0.5238095238095238	0.48119392684610074	0.827939135875841	0.9345662958086165
chr2	100401	.	T	C	14	5	19	0.7368421052631579	0.68	0.8063588989035531	0.9345662958086165
chr2	100501	.	A	C	18	8	26	0.6923076923076923	0.5005719474891224	0.07552363225182375	0.17694108127570135
chr2	100601	.	A	C	8	16	24	0.3333333333333333	0.5005719474891224	0.10735625822938202	0.23475235132824868
chr2	100901	.	C	A	13	5	18	0.7222222222222222	0.48119392684610074	0.05685407812147629	0.13734204831083996
chr2	101001	.	T	A	17	6	23	0.7391304347826086	0.5733836064069896	0.13978190182760725	0.28311605765436193
chr2	101301	.	G	A	13	8	21	0.6190476190476191	0.47317805383022776	0.19621506014171408	0.3656735211731944
chr2	101401	.	T	C	15	3	18	0.8333333333333334	0.68	0.21015270121537097	0.38294492221467596
chr2	101601	.	A	G	14	11	25	0.56	0.5337917521075416	0.843389096217532	0.9345662958086165
chr2	102001	.	A	C	14	11	25	0.56	0.5005719474891224	0.6900428327474561	0.844530034108827
chr2	102201	.	T	G	3	19	22	0.13636363636363635	0.48658578263841423	0.0009315057815830121	0.0058730380856551285
chr2	102301	.	G	A	12	8	20	0.6	0.47317805383022776	0.2719339660076273	0.44155614282426614
chr3	100101	.	A	T	19	3	22	0.8636363636363636	0.6020910973084886	0.014626652880080979	0.04525983155345812
chr3	100301	.	T	A	12	9	21	0.5714285714285714	0.5733836064069896	1.0	1.0
chr3	100501	.	T	C	22	4	26	0.8461538461538461	0.68	0.09081559038736321	0.20944436651319068
chr3	100701	.	T	A	12	7	19	0.631578947368421	0.5733836064069896	0.6512552703823019	0.835461220826978
chr3	100901	.	C	T	23	6	29	0.7931034482758621	0.5	0.002315700054168701	0.010704888237847222
chr3	101001	.	T	G	4	21	25	0.16	0.48658578263841423	0.0010027138195020952	0.0058730380856551285
chr3	101401	.	A	T	12	6	18	0.6666666666666666	0.6020910973084886	0.6383703614504076	0.835461220826978
chr3	101501	.	G	A	11	11	22	0.5	0.47317805383022776	0.8337051373912427	0.9345662958086165
chr3	101601	.	A	C	12	9	21	0.5714285714285714	0.5005719474891224	0.6636282112565921	0.8417275778387743
chr3	101701	.	G	T	18	4	22	0.8181818181818182	0.37896825396825395	6.0355430665882026e-05	0.0008998446026549684
chr3	101901	.	G	C	2	18	20	0.1	0.6179314888010541	2.3301313426164552e-06	9.064675675991145e-05
chr3	102217	.	C	T	15	2	17	0.8823529411764706	0.5	0.002349853515625	0.010704888237847222
chr3	102301	.	C	G	3	17	20	0.15	0.5488721804511278	0.00039502524184612485	0.003599118870153582
chr4	100281	.	G	A	16	2	18	0.8888888888888888	0.47317805383022776	0.00046566670321115293	0.004019438911927846
chr4	100401	.	C	A	4	13	17	0.23529411764705882	0.48119392684610074	0.05171889907109348	0.13448548933041968
chr4	100501	.	T	G	6	21	27	0.2222222222222222	0.48658578263841423	0.00639466800480879	0.02330501228419203
chr4	100601	.	G	C	14	9	23	0.6086956521739131	0.6179314888010541	1.0	1.0
chr4	100701	.	T	C	7	11	18	0.3888888888888889	0.68	0.011338332647469094	0.03718973108369863
chr4	100901	.	T	A	13	9	22	0.5909090909090909	0.5733836064069896	1.0	1.0
chr4	101101	.	G	C	2	19	21	0.09523809523809523	0.6179314888010541	9.808049646283443e-07	8.042600709952423e-05
chr4	101301	.	A	G	14	8	22	0.6363636363636364	0.5337917521075416	0.39634445547050573	0.6074812214688124
chr4	101501	.	T	A	6	18	24	0.25	0.5733836064069896	0.0016210169681686267	0.008575702670311445
chr4	101601	.	A	C	10	17	27	0.37037037037037035	0.5005719474891224	0.18443753127226964	0.34767534630634733
chr4	102301	.	A	G	17	5	22	0.7727272727272727	0.5337917521075416	0.03098136709041349	0.0891393719794353
chr4	102401	.	T	G	17	3	20	0.85	0.48658578263841423	0.0011793658526510523	0.00644719999449242
chr5	100401	.	G	T	5	16	21	0.23809523809523808	0.37896825396825395	0.260458521059239	0.43586936177260405
chr5	101201	.	G	C	15	5	20	0.75	0.6179314888010541	0.25800476551714485	0.43586936177260405
chr5	102201	.	T	G	9	15	24	0.375	0.48658578263841423	0.3116526946518195	0.4867718278371276
chr5	102301	.	G	T	6	18	24	0.25	0.37896825396825395	0.21416734864313228	0.3859719250271834
chr5	102401	.	T	G	19	3	22	0.8636363636363636	0.48658578263841423	0.0003718913654466061	0.003599118870153582
chr7	100301	.	G	A	5	19	24	0.20833333333333334	0.47317805383022776	0.012524237461094446	0.040274018502342926
chr7	100401	.	A	T	6	18	24	0.25	0.6020910973084886	0.0005872350923895684	0.004377570688722237
chr7	100801	.	G	A	10	18	28	0.35714285714285715	0.47317805383022776	0.25820365549846	0.43586936177260405
chr7	100901	.	A	C	11	12	23	0.4782608695652174	0.5005719474891224	0.8386376914942997	0.9345662958086165
chr7	101001	.	T	C	4	19	23	0.17391304347826086	0.68	8.255646688658487e-07	8.042600709952423e-05
chr7	101101	.	T	C	5	14	19	0.2631578947368421	0.68	0.00023490951046382626	0.00275179712257625
chr7	101201	.	T	G	6	11	17	0.35294117647058826	0.48658578263841423	0.33529082491849693	0.5187518423267311
chr7	101701	.	G	A	20	5	25	0.8	0.47317805383022776	0.0010680105988393838	0.0060397840761951364
chr7	101801	.	G	A	9	14	23	0.391304347826087	0.47317805383022776	0.5326996479564567	0.7459969084452445
chr7	101901	.	G	T	9	15	24	0.375	0.37896825396825395	1.0	1.0
chr7	102001	.	G	A	12	11	23	0.5217391304347826	0.47317805383022776	0.680474084651959	0.8417275778387743
chr7	102201	.	G	C	16	3	19	0.8421052631578947	0.6179314888010541	0.056946702958153156	0.13734204831083996
chr7	102301	.	G	A	7	17	24	0.2916666666666667	0.47317805383022776	0.10058658175511447	0.22439670903502104
chr8	100701	.	T	A	18	4	22	0.8181818181818182	0.5733836064069896	0.028695678270086594	0.08403734350525359
chr8	101101	.	C	A	18	4	22	0.8181818181818182	0.48119392684610074	0.00202025756052162	0.010040067876531687
chr8	101201	.	T	C	7	16	23	0.30434782608695654	0.68	0.00038525464804259953	0.003599118870153582
chr8	102401	.	C	G	12	7	19	0.631578947368421	0.5488721804511278	0.5001709652847587	0.7195441956728108
chr9	100601	.	T	G	24	4	28	0.8571428571428571	0.48658578263841423	7.630192709295235e-05	0.0010427930036036822
chr9	101001	.	A	C	8	9	17	0.47058823529411764	0.5005719474891224	0.8143192037906958	0.9345662958086165
chr9	101201	.	A	G	8	12	20	0.4	0.5337917521075416	0.2664136754962508	0.4400774376631748
chr9	102401	.	A	C	15	10	25	0.6	0.5005719474891224	0.4243638702630204	0.6384924286526178
chr10	100101	.	A	T	7	17	24	0.2916666666666667	0.6020910973084886	0.002768310640413654	0.01164110115455998
chr10	100501	.	A	G	5	12	17	0.29411764705882354	0.5337917521075416	0.05388150871850565	0.13542237117835118
chr10	100801	.	C	T	6	14	20	0.3	0.5	0.11531829833984375	0.2488447490491365
chr10	101101	.	C	A	9	12	21	0.42857142857142855	0.48119392684610074	0.6683522552498751	0.8417275778387743
chr10	101201	.	C	G	13	6	19	0.6842105263157895	0.5488721804511278	0.25886793359956733	0.43586936177260405
chr10	101501	.	G	T	8	16	24	0.3333333333333333	0.37896825396825395	0.8338212377465094	0.9345662958086165
chr10	101601	.	G	A	10	11	21	0.47619047619047616	0.47317805383022776	1.0	1.0
chr10	101701	.	A	C	9	16	25	0.36	0.5005719474891224	0.16815671306848406	0.3244435405086045
chr10	102201	.	A	C	10	13	23	0.43478260869565216	0.5005719474891224	0.540794059812977	0.7459969084452445
chr10	102401	.	A	G	11	11	22	0.5	0.5337917521075416	0.8321472623711359	0.9345662958086165
chr11	100801	.	A	C	14	8	22	0.6363636363636364	0.5005719474891224	0.28628570774135603	0.4558335540736154
chr11	101601	.	T	G	16	8	24	0.6666666666666666	0.48658578263841423	0.10125217358897291	0.22439670903502104
chr11	102201	.	A	C	5	15	20	0.25	0.5005719474891224	0.02643640152285422	0.07882854272269257
chr12	100101	.	A	C	3	18	21	0.14285714285714285	0.5005719474891224	0.0008446881904994825	0.005772035968413131
chr12	100201	.	C	T	11	7	18	0.6111111111111112	0.5	0.480682373046875	0.7162942426320567
chr12	100301	.	A	T	12	8	20	0.6	0.6020910973084886	1.0	1.0
chr12	100701	.	T	C	16	5	21	0.7619047619047619	0.68	0.4916201857842766	0.7162942426320567
chr12	100882	.	G	C	14	2	16	0.875	0.6179314888010541	0.0387466249008695	0.10417125383184586
chr12	100901	.	T	A	9	8	17	0.5294117647058824	0.5733836064069896	0.8078720958511445	0.9345662958086165
chr12	101001	.	C	A	11	8	19	0.5789473684210527	0.48119392684610074	0.4928335733393132	0.7162942426320567
chr12	101601	.	C	T	17	6	23	0.7391304347826086	0.5	0.03468966484069824	0.09808801782542262
chr12	102401	.	C	T	17	3	20	0.85	0.5	0.0025768280029296875	0.011121047170538651
chr16	100101	.	G	T	14	7	21	0.6666666666666666	0.37896825396825395	0.01127940118578678	0.03718973108369863
chr16	100201	.	A	G	20	5	25	0.8	0.5337917521075416	0.008267377851739739	0.029474999297506892
chr16	100256	.	C	G	18	3	21	0.8571428571428571	0.5488721804511278	0.0039085823899731404	0.01536929011331926
chr16	100301	.	G	T	6	13	19	0.3157894736842105	0.37896825396825395	0.64341965617564	0.835461220826978
chr16	100701	.	G	A	4	13	17	0.23529411764705882	0.47317805383022776	0.05449924693762914	0.13542237117835118
chr16	100801	.	T	A	16	5	21	0.7619047619047619	0.5733836064069896	0.12041628418907602	0.24997810894947428
chr16	102101	.	C	G	21	7	28	0.75	0.5488721804511278	0.03641600845840431	0.10122415910471706
chr16	102201	.	C	A	10	13	23	0.43478260869565216	0.48119392684610074	0.6826205356863231	0.8417275778387743
chr18	100301	.	A	T	17	8	25	0.68	0.6020910973084886	0.5413026347864883	0.7459969084452445
chr18	100501	.	T	C	17	8	25	0.68	0.68	1.0	1.0
chr18	100601	.	C	T	21	3	24	0.875	0.5	0.0002771615982055664	0.0030303001403808594
chr18	100701	.	A	T	14	5	19	0.7368421052631579	0.6020910973084886	0.2529682837256748	0.43586936177260405
chr18	101301	.	T	A	16	5	21	0.7619047619047619	0.5733836064069896	0.12041628418907602	0.24997810894947428
chr18	101401	.	A	G	17	8	25	0.68	0.5337917521075416	0.16352693153905024	0.3192668663381457
chr18	101501	.	A	C	4	21	25	0.16	0.5005719474891224	0.0005262902550106592	0.004154467855525064
chr18	101801	.	T	G	14	8	22	0.6363636363636364	0.48658578263841423	0.20126039114230276	0.37086184435210845
chr18	102301	.	A	G	5	14	19	0.2631578947368421	0.5337917521075416	0.021011962197957063	0.06381410741601774
chr19	100901	.	C	G	21	4	25	0.84	0.5488721804511278	0.003936037711947615	0.01536929011331926
chr20	100101	.	T	A	13	5	18	0.7222222222222222	0.5733836064069896	0.23942606815081918	0.42680299105146025
chr20	101301	.	T	A	5	14	19	0.2631578947368421	0.5733836064069896	0.00902598285947566	0.031494918913915065
chr20	102201	.	A	C	10	11	21	0.47619047619047616	0.5005719474891224	0.8316217186283126	0.9345662958086165
chr20	102401	.	G	C	22	8	30	0.7333333333333333	0.6179314888010541	0.25934439477335425	0.43586936177260405
