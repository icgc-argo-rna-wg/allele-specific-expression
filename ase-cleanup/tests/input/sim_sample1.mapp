contig	pos	mappability
chr1	41	.
chr1	72	.
chr1	401	.
chr1	701	.
chr1	1001	.
chr1	1101	.
chr1	1201	.
chr1	1401	.
chr1	1501	.
chr1	1567	.
chr1	1646	.
chr1	1701	.
chr1	1801	.
chr1	2001	.
chr1	2601	.
chr1	2701	.
chr1	2801	.
chr1	3001	.
chr1	3301	.
chr1	3601	.
chr1	3801	.
chr1	3992	.
chr1	4073	.
chr1	4201	.
chr1	4701	.
chr1	4801	.
chr1	4901	.
chr1	5001	.
chr1	5201	.
chr1	5401	.
chr1	5504	.
chr1	5601	.
chr1	5701	.
chr1	5801	.
chr1	6301	.
chr1	6401	.
chr1	6501	.
chr1	6901	.
chr1	7101	.
chr1	7201	.
chr1	7301	.
chr1	7701	.
chr1	7801	.
chr1	7901	.
chr1	8201	.
chr1	8301	.
chr1	8601	.
chr1	9001	.
chr1	9101	.
chr1	9301	.
chr1	9601	.
chr1	9801	.
chr1	10001	.
chr1	10101	0.1
chr1	10501	.
chr1	10701	.
chr1	10731	.
chr1	11201	.
chr1	11301	.
chr1	11374	.
chr1	11501	.
chr1	11701	.
chr1	11801	.
chr1	11901	.
chr1	12101	.
chr1	12300	0.56
chr1	12701	.
chr1	12839	.
chr1	13301	0.48
chr1	13401	.
chr1	13701	.
chr1	13794	.
chr1	13901	.
chr1	14101	.
chr1	14601	.
chr1	15101	.
chr1	15116	.
chr1	15501	.
chr1	15601	.
chr1	15701	.
chr1	15801	0.14
chr1	15838	.
chr1	16001	.
chr1	16101	0.92
chr1	16308	0.8
chr1	16601	.
chr1	16877	.
chr1	16901	.
chr1	17101	.
chr1	17301	.
chr1	17401	.
chr1	17701	.
chr1	18223	1
chr1	18401	.
chr1	18701	.
chr1	18801	0.04
chr1	18901	.
chr1	19101	.
chr10	501	.
chr10	601	.
chr10	862	.
chr10	901	.
chr10	1001	.
chr10	1201	.
chr10	1601	.
chr10	1901	.
chr10	2001	.
chr10	2101	.
chr10	2601	.
chr10	3101	.
chr10	3180	.
chr10	3201	.
chr10	3301	.
chr10	3601	.
chr10	3701	.
chr10	4201	.
chr10	4401	.
chr10	4601	.
chr10	5101	.
chr10	5158	.
chr10	5201	.
chr10	5640	.
chr10	5801	.
chr10	6006	.
chr10	6301	.
chr10	6401	.
chr10	6801	.
chr10	7001	.
chr10	7101	.
chr10	7201	.
chr10	7301	.
chr10	7601	.
chr10	7801	.
chr10	7931	.
chr10	8001	.
chr10	8101	.
chr10	8301	.
chr10	8601	.
chr10	8901	.
chr10	9501	.
chr10	9701	.
chr10	10201	.
chr10	10301	.
chr10	10401	.
chr10	10501	.
chr10	10701	.
chr10	10801	.
chr11	58	.
chr11	301	.
chr11	401	.
chr11	501	.
chr11	601	.
chr11	1601	.
chr11	1701	.
chr11	1801	.
chr11	1901	.
chr11	2001	.
chr11	2328	.
chr11	2588	.
chr11	2601	.
chr11	2901	.
chr11	3301	.
chr11	3801	.
chr11	4101	.
chr11	4601	.
chr11	4701	.
chr11	4801	.
chr11	5201	.
chr11	5501	.
chr11	5533	.
chr11	5901	.
chr11	6501	.
chr11	6601	.
chr11	6801	.
chr11	6901	.
chr11	6995	.
chr11	7601	.
chr11	7801	.
chr11	8001	.
chr11	8101	.
chr11	8201	.
chr11	8401	.
chr11	8402	.
chr11	8901	.
chr11	9001	.
chr11	9355	.
chr11	9401	.
chr11	9501	.
chr11	9801	.
chr11	9901	.
chr11	10101	.
chr11	10301	.
chr11	10501	.
chr11	10701	.
chr11	10801	.
chr11	10863	.
chr11	10901	.
chr11	11201	.
chr12	101	.
chr12	301	.
chr12	801	.
chr12	901	.
chr12	1001	.
chr12	1301	.
chr12	1401	.
chr12	1501	.
chr12	1553	.
chr12	1901	.
chr12	2301	.
chr12	2401	.
chr12	2559	.
chr12	2601	.
chr12	2701	.
chr12	3101	.
chr12	3801	.
chr12	3901	.
chr12	4201	.
chr12	4301	.
chr12	5001	.
chr12	5101	.
chr12	5501	.
chr12	5601	.
chr12	6001	.
chr12	6501	.
chr12	6601	.
chr12	6701	.
chr12	6901	.
chr12	7101	.
chr13	301	.
chr13	401	.
chr13	753	.
chr13	1401	.
chr13	1578	.
chr13	1801	.
chr13	1921	.
chr13	2101	.
chr13	2501	.
chr13	2701	.
chr13	2801	.
chr13	3001	.
chr13	3101	.
chr13	3201	.
chr13	3601	.
chr13	3765	.
chr13	3801	.
chr13	4501	.
chr13	4601	.
chr13	4668	.
chr13	4801	.
chr13	5101	.
chr13	5201	.
chr13	5401	.
chr13	5501	.
chr13	5901	.
chr13	6101	.
chr13	6201	.
chr13	6301	.
chr13	6417	.
chr13	6501	.
chr13	6701	.
chr14	501	.
chr14	701	.
chr14	901	.
chr14	1101	.
chr14	1201	.
chr14	1301	.
chr14	1401	.
chr14	1801	.
chr14	1901	.
chr14	2001	.
chr14	2101	.
chr14	2401	.
chr14	2601	.
chr14	2701	.
chr14	2901	.
chr14	3001	.
chr14	3301	.
chr14	3401	.
chr14	3601	.
chr14	3701	.
chr14	4201	.
chr14	4401	.
chr14	5101	.
chr14	5201	.
chr14	5301	.
chr14	5401	.
chr14	5701	.
chr14	6001	.
chr14	6101	.
chr14	6801	.
chr14	7001	.
chr14	7101	.
chr14	7401	.
chr14	7601	.
chr14	7801	.
chr14	7901	.
chr14	8501	.
chr14	8801	.
chr14	9001	.
chr14	9201	.
chr14	9301	.
chr14	9401	.
chr14	9701	.
chr14	10101	.
chr14	10501	.
chr14	11001	.
chr14	11101	.
chr14	11501	.
chr14	11701	.
chr14	11901	.
chr14	12601	.
chr14	12801	.
chr14	12901	.
chr14	13001	.
chr14	13201	.
chr14	13501	.
chr14	13801	.
chr14	13901	.
chr14	14016	.
chr14	14201	.
chr14	14301	.
chr14	14501	.
chr14	14601	.
chr14	14701	.
chr14	14801	.
chr14	15051	.
chr14	15301	.
chr14	15501	.
chr15	401	.
chr15	501	.
chr15	1101	.
chr15	1201	.
chr15	1401	.
chr15	1801	.
chr15	2101	.
chr15	2201	.
chr15	2401	.
chr15	2501	.
chr15	2801	.
chr15	2901	.
chr15	3009	.
chr15	3125	.
chr15	3267	.
chr15	3501	.
chr15	3901	.
chr15	4001	.
chr15	4101	.
chr15	4301	.
chr15	4401	.
chr15	4501	.
chr15	4901	.
chr15	5201	.
chr15	5401	.
chr15	5601	.
chr15	5801	.
chr15	6101	.
chr15	6201	.
chr15	6601	.
chr15	7196	.
chr15	7201	.
chr15	7401	.
chr15	7825	.
chr15	8301	.
chr15	8401	.
chr15	8501	.
chr15	8601	.
chr15	8701	.
chr15	8901	.
chr15	9101	.
chr15	9301	.
chr15	9601	.
chr15	9801	.
chr15	10101	.
chr15	10501	.
chr15	10801	.
chr15	11001	.
chr15	11098	.
chr15	11101	.
chr15	11401	.
chr15	12201	.
chr15	12347	.
chr15	12401	.
chr15	12501	.
chr15	12701	.
chr16	201	.
chr16	342	.
chr16	501	.
chr16	801	.
chr16	901	.
chr16	1601	.
chr16	1801	.
chr16	1874	.
chr16	2301	.
chr16	2401	.
chr16	2701	.
chr16	2801	.
chr16	3001	.
chr16	3401	.
chr16	3501	.
chr16	4401	.
chr16	4501	.
chr16	4601	.
chr16	4701	.
chr16	4901	.
chr16	5401	.
chr16	5601	.
chr16	5801	.
chr16	6201	.
chr16	6222	.
chr16	6401	.
chr16	6601	.
chr16	6701	.
chr16	6901	.
chr16	7201	.
chr16	7701	.
chr16	7801	.
chr16	8001	.
chr16	8101	.
chr16	8201	.
chr16	8301	.
chr16	8801	.
chr16	9201	.
chr16	9301	.
chr16	9901	.
chr16	10401	.
chr16	10501	.
chr16	10604	.
chr16	11001	.
chr16	12301	.
chr16	12501	.
chr16	13101	.
chr17	117	.
chr17	501	.
chr17	1501	.
chr17	1601	.
chr17	1701	.
chr17	1901	.
chr17	2449	.
chr17	3001	.
chr17	3431	.
chr17	3601	.
chr17	3701	.
chr17	3801	.
chr17	3992	.
chr17	4301	.
chr17	4654	.
chr17	4701	.
chr17	4801	.
chr17	4901	.
chr17	5101	.
chr17	5155	.
chr17	5201	.
chr17	5601	.
chr17	5796	.
chr17	6101	.
chr17	6301	.
chr17	6601	.
chr17	6701	.
chr17	7001	.
chr17	7101	.
chr17	7601	.
chr17	7801	.
chr17	7886	.
chr17	8001	.
chr17	8801	.
chr17	9401	.
chr17	9601	.
chr17	9701	.
chr17	10901	.
chr17	11001	.
chr17	11701	.
chr17	11801	.
chr17	11901	.
chr17	12001	.
chr17	12025	.
chr17	12601	.
chr17	13008	.
chr17	13101	.
chr17	13201	.
chr18	101	.
chr18	301	.
chr18	501	.
chr18	601	.
chr18	801	.
chr18	1201	.
chr18	1301	.
chr18	1501	.
chr18	1901	.
chr18	2001	.
chr18	2101	.
chr19	101	.
chr19	401	.
chr19	701	.
chr19	901	.
chr19	1601	.
chr19	2001	.
chr19	2101	.
chr19	2301	.
chr19	2601	.
chr19	2762	.
chr19	3001	.
chr19	3201	.
chr19	3401	.
chr19	3501	.
chr19	3701	.
chr19	3801	.
chr19	4001	.
chr19	4280	.
chr19	4301	.
chr19	4801	.
chr19	5201	.
chr19	5301	.
chr19	5501	.
chr19	5601	.
chr19	5701	.
chr19	5901	.
chr19	6001	.
chr19	6201	.
chr19	6301	.
chr19	6401	.
chr19	6601	.
chr19	6901	.
chr19	7101	.
chr19	7301	.
chr19	7401	.
chr19	7601	.
chr19	7901	.
chr19	8205	.
chr19	8362	.
chr19	8601	.
chr19	8657	.
chr19	8842	.
chr19	9101	.
chr19	9301	.
chr19	9401	.
chr19	9501	.
chr19	9601	.
chr19	9901	.
chr19	10201	.
chr19	10501	.
chr19	10601	.
chr19	10901	.
chr19	11101	.
chr19	11201	.
chr19	11501	.
chr19	11901	.
chr19	12001	.
chr19	12045	.
chr19	12101	.
chr19	12347	.
chr19	12434	.
chr19	12901	.
chr19	13001	.
chr19	13301	.
chr19	13401	.
chr19	13569	.
chr19	13701	.
chr19	13901	.
chr19	14395	.
chr19	14401	.
chr19	14601	.
chr19	14701	.
chr19	15101	.
chr19	15401	.
chr19	15501	.
chr19	15601	.
chr19	15701	.
chr19	15896	.
chr19	16201	.
chr19	16301	.
chr19	16801	.
chr19	16901	.
chr19	17001	.
chr19	17301	.
chr19	17601	.
chr19	18001	.
chr19	18301	.
chr19	18401	.
chr19	18601	.
chr19	18801	.
chr19	19501	.
chr19	19601	.
chr19	20001	.
chr19	20101	.
chr19	20201	.
chr19	20594	.
chr19	20601	.
chr19	20901	.
chr19	21001	.
chr19	21301	.
chr2	301	.
chr2	801	.
chr2	901	.
chr2	1201	.
chr2	1301	.
chr2	1701	.
chr2	1801	.
chr2	2201	.
chr2	2301	.
chr2	2901	.
chr2	3201	.
chr2	3301	.
chr2	3901	.
chr2	4001	.
chr2	4501	.
chr2	4801	.
chr2	5001	.
chr2	5350	.
chr2	5701	.
chr2	5901	.
chr2	6101	.
chr20	301	.
chr20	501	.
chr20	701	.
chr20	901	.
chr20	1001	.
chr20	1101	.
chr20	1501	.
chr20	1804	.
chr20	1901	.
chr20	2101	.
chr20	2201	.
chr20	2601	.
chr20	3201	.
chr20	3301	.
chr20	3701	.
chr20	3901	.
chr20	4001	.
chr20	4201	.
chr20	4401	.
chr20	4801	.
chr20	4822	.
chr20	5201	.
chr20	5301	.
chr20	5401	.
chr20	5801	.
chr20	6001	.
chr20	6101	.
chr20	6201	.
chr20	6401	.
chr20	6501	.
chr20	6601	.
chr20	6701	.
chr20	7201	.
chr20	7301	.
chr20	8001	.
chr20	8101	.
chr20	9001	.
chr20	9201	.
chr20	9901	.
chr20	10101	.
chr20	10201	.
chr20	10301	.
chr20	10401	.
chr20	10501	.
chr20	10801	.
chr20	10901	.
chr20	11001	.
chr20	11201	.
chr20	11441	.
chr20	11501	.
chr20	11530	.
chr20	11601	.
chr20	11901	.
chr20	12201	.
chr20	12301	.
chr20	12901	.
chr20	13301	.
chr20	13401	.
chr20	13801	.
chr20	13945	.
chr20	14201	.
chr20	14401	.
chr20	14601	.
chr20	14801	.
chr20	15201	.
chr20	15301	.
chr20	15401	.
chr20	15701	.
chr20	16101	.
chr20	16201	.
chr20	16301	.
chr20	16501	.
chr20	16801	.
chr20	16901	.
chr20	17001	.
chr20	17301	.
chr20	17901	.
chr20	18101	.
chr20	18501	.
chr20	18701	.
chr20	18901	.
chr20	18902	.
chr20	19001	.
chr20	19401	.
chr20	19701	.
chr21	85	.
chr21	147	.
chr21	601	.
chr21	801	.
chr21	901	.
chr21	1001	.
chr21	1401	.
chr21	1643	.
chr21	1901	.
chr21	2301	.
chr21	2801	.
chr21	2901	.
chr21	3001	.
chr21	3101	.
chr21	3201	.
chr21	3601	.
chr21	4001	.
chr21	4301	.
chr21	4501	.
chr21	4601	.
chr21	4701	.
chr21	5001	.
chr21	5401	.
chr21	5501	.
chr21	5701	.
chr21	5901	.
chr21	6101	.
chr21	6325	.
chr21	6693	.
chr21	6801	.
chr21	6901	.
chr22	101	.
chr22	201	.
chr22	412	.
chr22	605	.
chr22	1101	.
chr22	1501	.
chr22	1581	.
chr22	1701	.
chr22	1901	.
chr22	2020	.
chr22	2201	.
chr22	2301	.
chr22	2401	.
chr22	2801	.
chr22	2901	.
chr22	3201	.
chr22	3401	.
chr22	3601	.
chr22	3701	.
chr22	4001	.
chr22	4301	.
chr22	4501	.
chr22	4601	.
chr22	5501	.
chr22	5801	.
chr22	6001	.
chr3	701	.
chr3	801	.
chr3	901	.
chr3	1038	.
chr3	1201	.
chr3	1301	.
chr3	1401	.
chr3	1901	.
chr3	2201	.
chr3	2401	.
chr3	2601	.
chr3	2693	.
chr3	2701	.
chr3	2801	.
chr3	3201	.
chr3	3401	.
chr3	3601	.
chr3	3801	.
chr3	4601	.
chr3	4901	.
chr3	4988	.
chr3	5501	.
chr3	5901	.
chr3	6001	.
chr3	6101	.
chr3	6201	.
chr3	6401	.
chr3	6501	.
chr3	6624	.
chr3	6701	.
chr3	7101	.
chr3	7201	.
chr3	7391	.
chr3	7401	.
chr3	7601	.
chr3	7801	.
chr3	8001	.
chr3	8163	.
chr3	8301	.
chr3	8401	.
chr3	8501	.
chr3	9101	.
chr3	9201	.
chr3	9301	.
chr3	9401	.
chr3	9501	.
chr3	10201	1
chr3	10601	0.96
chr3	10801	1
chr3	11201	.
chr3	11501	.
chr3	11801	1
chr3	12101	1
chr3	12201	0.5
chr3	12701	1
chr3	12801	0.98
chr3	13301	0.94
