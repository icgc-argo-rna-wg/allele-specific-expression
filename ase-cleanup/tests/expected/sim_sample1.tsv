contig	position	variantID	refAllele	altAllele	refCount	altCount	totalCount	ase_ratio	ref_bias	AEI_pval	AEI_padj	gene_id	feature
chr1	10101	.	T	G	6	12	18	0.3333333333333333	0.5008743826896312	0.1663364886098673	0.3161366011591044		
chr1	12300	.	C	A	12	2	14	0.8571428571428571	0.4723686819255536	0.0053388581775967	0.0301761549168509	ENSG00000223972	intron
chr1	13301	.	A	C	10	3	13	0.7692307692307693	0.5365817512614312	0.1035073776404184	0.2292326932411311	ENSG00000223972	exon
chr1	15801	.	T	G	6	8	14	0.4285714285714285	0.5008743826896312	0.6066050106951036	0.7889250364567292	ENSG00000227232	exon
chr1	16101	.	C	T	8	6	14	0.5714285714285714	0.5266952326073545	0.7946715431247644	0.9055634606411188	ENSG00000227232	intron
chr1	16308	.	G	C	18	2	20	0.9	0.5219634137845988	0.000499448656267	0.0132506786356555	ENSG00000227232	intron
chr1	18223	.	A	T	17	2	19	0.8947368421052632	0.5295984682136045	0.0010121688805856	0.015683948867577	ENSG00000227232	intron
chr3	10201	.	T	G	7	9	16	0.4375	0.5008743826896312	0.6284563275686177	0.7965934966362904		
chr3	10601	.	C	A	5	15	20	0.25	0.4723686819255536	0.0705557690689448	0.1709940722587086		
chr3	10801	.	C	T	9	7	16	0.5625	0.5266952326073545	0.8080532879395532	0.9055634606411188		
chr3	11801	.	T	G	17	4	21	0.8095238095238095	0.5008743826896312	0.0071991605456129	0.0365582371456908	ENSG00000223587	exon
chr3	12101	.	T	C	10	10	20	0.5	0.5152445034664859	1.0	1.0	ENSG00000223587	exon
chr3	12201	.	C	T	3	14	17	0.1764705882352941	0.5266952326073545	0.0057076125453676	0.0307879514895349	ENSG00000223587	exon
chr3	12701	.	C	T	2	15	17	0.1176470588235294	0.5266952326073545	0.00086600927184	0.0146145005183963	ENSG00000223587	intron
chr3	12801	.	A	T	12	5	17	0.7058823529411765	0.5295984682136045	0.2234819840410414	0.3913405710658321	ENSG00000223587	intron
chr3	13301	.	G	T	12	5	17	0.7058823529411765	0.4947547970571774	0.0930765729520938	0.2111684901181884	ENSG00000223587	intron
